packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -157,9 +157,8 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
   }
 
   if (isRecording && !isFlashingFreshness) {
-    // During face matching, we want to only show the TOO_CLOSE or
-    // TOO_FAR texts. If FaceMatchState matches TOO_CLOSE, we'll show
-    // the TOO_CLOSE text, but for FACE_IDENTIFED, CANT_IDENTIFY, TOO_MANY
+    // During face matching, we want to only show the
+    // TOO_FAR texts. For FACE_IDENTIFIED, CANT_IDENTIFY, TOO_MANY
```

Comment:
We still want to show TOO_CLOSE if they start too close, does this PR account for that?

Reply:
Yes this does, it will tell you to move back only before recording starts